CSSS/Kernel.cs

```diff
@@ -166,6 +171,32 @@ public bool PerformTasks()
                 PerformIssueCheckTasks();
             }
 
+            // If we should shutdown
+            if (shouldShutdown)
+            {
+                // Run the shutdown command
+                var process = new System.Diagnostics.Process();
+                process.StartInfo.FileName = "shutdown";
+                switch (config.operatingSystemType)
+                {
+                    case Config.OperatingSystemType.WinNT:
+                        // On Windows, we can specify the seconds
+                        // to delay the shutdown :D
+                        // So we shutdown in 15 seconds (should be suffficient time)
+                        process.StartInfo.Arguments = "-s -t 15";

```

Comment:
What is the "_sufficient time_" for? Could the shutdown timeout instead be 1 second?

Reply:
I will lower the shutdown time on Windows to 5 seconds. However, the reasoning behind this is that it would be better for CSSS to quit of its own accord than be forcefully closed by the computer shutting down.